src/Middleware/WebSockets/test/UnitTests/WebSocketMiddlewareTests.cs

```diff
@@ -498,14 +498,17 @@ public async Task CloseFromCloseReceived_Success()
     }
 
     [Fact]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/48933")]
     public async Task WebSocket_Abort_Interrupts_Pending_ReceiveAsync()
     {
         WebSocket serverSocket = null;
 
+        // Events that we want to sequence execution across client and server.
         var socketWasAccepted = new ManualResetEventSlim();
         var socketWasAborted = new ManualResetEventSlim();
         var firstReceiveOccured = new ManualResetEventSlim();
+        var secondReceiveInitiated = new ManualResetEventSlim();
```

Comment:
ManualResetEventSlim instead of TaskCompletionSource? How old fashion 😋

Reply:
Hipster code.